### DISCUSSION TITLE
v2.19

### DISCUSSION THREAD

**Discussion by Ralim** [Ralim/IronOS#1329] [General]
This is a smaller release going out, mostly fixes for Logo's and the MHP30.

## Key changes

* Bug-fix Infinite Boot Logo
* Shutdown settings for MHP30
* Accelerometer sensitivity for MHP30
* Allow showing unique device ID
* Bug-fix chance of a power pulse at device boot

## What's Changed (Detailed)
* Update translation_RU.json by @JugglerLKR in https://github.com/Ralim/IronOS/pull/1280
* Update docs with new Miniware DFU bootloader by @cewbdex in https://github.com/Ralim/IronOS/pull/1278
* Minor Bootup Logo readme fix by @LyricPants66133 in https://github.com/Ralim/IronOS/pull/1281
* Update translation_BE.json by @dmitrygribenchuk in https://github.com/Ralim/IronOS/pull/1282
* Typo fixes and various improvements by @cewbdex in https://github.com/Ralim/IronOS/pull/1279
* Update README.md by @River-b in https://github.com/Ralim/IronOS/pull/1284
* Update PL translation (v2.18) by @KITT3000 in https://github.com/Ralim/IronOS/pull/1286
* Update translation_BE.json by @dmitrygribenchuk in https://github.com/Ralim/IronOS/pull/1287
* Update Czech translation by @cewbdex in https://github.com/Ralim/IronOS/pull/1290
* Update translation_BE.json by @dmitrygribenchuk in https://github.com/Ralim/IronOS/pull/1291
* removing some redundant lines by @discip in https://github.com/Ralim/IronOS/pull/1293
* Add USB autosuspend to troubleshooting for Pinecil by @BjoernLange in https://github.com/Ralim/IronOS/pull/1302
* Fix for slang expressions by @onxblog in https://github.com/Ralim/IronOS/pull/1289
* Update dutch translation by @FaBjE in https://github.com/Ralim/IronOS/pull/1297
* Update translation_RU.json by @JugglerLKR in https://github.com/Ralim/IronOS/pull/1304
* removed some more redundant entries by @discip in https://github.com/Ralim/IronOS/pull/1312
* Infinite logo by @Ralim in https://github.com/Ralim/IronOS/pull/1309
* Startup behaviour fixes by @Ralim in https://github.com/Ralim/IronOS/pull/1306
* MHP30: Shutdown settings by @Ralim in https://github.com/Ralim/IronOS/pull/1307
* DeviceID by @Ralim in https://github.com/Ralim/IronOS/pull/1314
* Update translation_PL.json by @d-rez in https://github.com/Ralim/IronOS/pull/1320
* Adding features table by @Ralim in https://github.com/Ralim/IronOS/pull/1318
* Update translation_FR.json by @HanaO00 in https://github.com/Ralim/IronOS/pull/1322
* more obvious feature list by @discip in https://github.com/Ralim/IronOS/pull/1325
* Release 2.19 by @Ralim in https://github.com/Ralim/IronOS/pull/1324

## New Contributors
* @cewbdex made their first contribution in https://github.com/Ralim/IronOS/pull/1278
* @LyricPants66133 made their first contribution in https://github.com/Ralim/IronOS/pull/1281
* @dmitrygribenchuk made their first contribution in https://github.com/Ralim/IronOS/pull/1282
* @BjoernLange made their first contribution in https://github.com/Ralim/IronOS/pull/1302
* @FaBjE made their first contribution in https://github.com/Ralim/IronOS/pull/1297
* @d-rez made their first contribution in https://github.com/Ralim/IronOS/pull/1320

**Full Changelog**: https://github.com/Ralim/IronOS/compare/v2.18.2...v2.19

<hr /><em>This discussion was created from the release <a href='https://github.com/Ralim/IronOS/releases/tag/v2.19'>v2.19</a>.</em>

**Comment by hasechris** [comment]
Hi @Ralim,

where is the download for pinecil v2?

Best regards
hasechris

**Reply by Ralim** [reply]
See the 2.20 betas on the PR for 2.20

**Reply by River-Mochi** [reply]
> where is the download for pinecil v2?

I recommend getting the PR for BLE, it works for my EPR chargers 28V usb-C  and allows BLE bluetooth testings also with  joric's beta API webpage.
